modules/integration/src/test/scala/scala/cli/integration/PublishTestDefinitions.scala

```diff
@@ -271,4 +273,212 @@ abstract class PublishTestDefinitions(val scalaVersionOpt: Option[String])
       expect(mainClasses == Set(scalaFile1, scalaFile2, s"$scriptsDir.${scriptName}_sc"))
     }
   }
+
+  test("missing secret key password") {
+    // format: off
+    val signingOptions = Seq(
+      "--secret-key", s"file:key.skr",
+      "--signer", "bc"
+    )
+    // format: on
+
+    TestCase.testInputs.fromRoot { root =>
+      os.proc(
+        TestUtil.cli,
+        "--power",
+        "config",
```

Comment:
Doesn't this change the global user config? Better set `SCALA_CLI_CONFIG` when running commands in this test (look for `SCALA_CLI_CONFIG` in `RunScalaNativeTestDefinitions` or `ConfigTests` for example), so that the config changes apply only to the tests, and the global config is ignored.

Reply:
Alternatively, we could consider changing the default config directory for integration tests. It could be handled around [here](https://github.com/VirtusLab/scala-cli/blob/0d4c5f762010c13172b5c413f662f63cfa66c803/build.sc#L870).

Even if we forget to set the `SCALA_CLI_CONFIG` in tests, any changes in `config` would not affect the global user config.